src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/IssueApplicationContestedSubmittedHandler.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.handler;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.finrem.caseorchestration.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.controllers.GenericAboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.finrem.caseorchestration.mapper.FinremCaseDetailsMapper;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.EventType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseData;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.correspondence.issueapplication.IssueApplicationContestedEmailCorresponder;
+
+@Slf4j
+@Service
+public class IssueApplicationContestedSubmittedHandler extends FinremCallbackHandler {
+
+    private final IssueApplicationContestedEmailCorresponder corresponder;
+
+    public IssueApplicationContestedSubmittedHandler(FinremCaseDetailsMapper finremCaseDetailsMapper,
+                                                     IssueApplicationContestedEmailCorresponder corresponder) {
+        super(finremCaseDetailsMapper);
+        this.corresponder = corresponder;
+    }
+
+    @Override
+    public boolean canHandle(CallbackType callbackType, CaseType caseType, EventType eventType) {
+        return CallbackType.SUBMITTED.equals(callbackType)
+            && CaseType.CONTESTED.equals(caseType)
+            && (EventType.ISSUE_APPLICATION.equals(eventType));
+    }
+
+
+    @Override
+    public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(FinremCallbackRequest callbackRequest,
+                                                                              String userAuthorisation) {
+
+        Long caseId = callbackRequest.getCaseDetails().getId();
+        log.info("Received request to send email for contested 'Application Issued' for Case ID: {}", caseId);
```

Comment:
We have logs in correspondence
e.g. log.info("Sending email correspondence to applicant for case: {}", caseDetails.getId());